controller/linodemachine_controller_helpers_test.go

```diff
@@ -61,3 +75,295 @@ func TestLinodeMachineSpecToCreateInstanceConfig(t *testing.T) {
 
 	assert.Equal(t, machineSpec, actualMachineSpec)
 }
+
+func TestSetUserData(t *testing.T) {
```

Comment:
Should we disable go-analyse on test files?

Reply:
I've added the failing check as one to ignore for test files, I'm hoping that solves our issue here